src/D2L.CodeStyle.Analyzers/Immutability/UnsafeStaticsAnalyzer.cs

```diff
@@ -75,20 +79,19 @@ public sealed class UnsafeStaticsAnalyzer : DiagnosticAnalyzer {
 				InspectFieldOrProperty(
 					context,
 					inspector,
+					fieldOrProperty: symbol,
 					location: variable.GetLocation(),
 					isStatic: isStatic,
-					isReadOnly: isReadOnly,
-					fieldOrProperty: symbol,
 					fieldOrPropertyType: symbol.Type,
-					fieldOrPropertyName: variable.Identifier.ValueText,
-					initializer: variable.Initializer?.Value,
-					isProperty: false,
-					isAutoImplementedProperty: false
+					fieldOrPropertyName: variable.Identifier.ValueText

```

Comment:
A lot of things don't need to be passed down anymore, but I also tried to consistently sort the arguments in the different functions so some of the changes are just things moving around.